ArjunaJTA/jta/pom.xml

```diff
@@ -25,8 +25,9 @@
   <description>Narayana: ArjunaJTA jta</description>
 
   <properties>
+    <!-- java.net.preferIPv6Addresses=false is a workaround to (not) solve the bug in arquillian-extension-byteman -->
     <ipv6.server.jvm.args>-Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1]
-      -Djboss.bind.address.unsecure=[::1] -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true</ipv6.server.jvm.args>
+      -Djboss.bind.address.unsecure=[::1] -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=false</ipv6.server.jvm.args>
```

Comment:
Could I ask you why did you set it to false? What bug does it solve?

Reply:
This is a workaround for a bug (related to addresses) in `arquillian-extension-byteman`

Reply:
I understand, did you manage to make the arqIPv6 profile work after that? In my local environment that profile does not work (before and after setting that property to false).

Reply:
Well, I thought I did (several times). Maybe someone else should double check if this PR passes arqIPv6?

Reply:
(FYI I tested it again and the arqIPv6 profile passes locally)

Reply:
Thank you for confirming Manuel. Then I think it is a problem of my environment.